modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -1512,13 +1547,26 @@ public final BlendMode getBlendMode() {
     public final ObjectProperty<BlendMode> blendModeProperty() {
         if (blendMode == null) {
             blendMode = new StyleableObjectProperty<BlendMode>(null) {
+                static final CssMetaData<Node, BlendMode> METADATA = new CssMetaData<>(
+                        "-fx-blend-mode", new EnumConverter<>(BlendMode.class)) {
+                    @Override
+                    public boolean isSettable(Node node) {
+                        return node.blendMode == null || !node.blendMode.isBound();
+                    }
+
+                    @Override
+                    public StyleableProperty<BlendMode> getStyleableProperty(Node node) {
+                        return (StyleableProperty<BlendMode>)node.blendModeProperty();
```

Comment:
these two methods can be implemented in the base class if we add a
`setGetter(Function<S, StyleableProperty<V>> getter)`
and also
`setIsSettable(Predicate<S>)`
?

This way we don't even need to declare a new class each time.

Just a thought.

Reply:
Yes, this can be further simplified.